server/server.js

```diff
@@ -1,5 +1,5 @@
 import { Strategy as SlackStrategy } from "passport-slack";
-import { notifHandler, oAuthHandler } from "hull/lib/utils";
```

Comment:
@mkawalek for transition period we should have both, notifHandler on current endpoint, smart-notifier on new one.